kudu/tsconfig.json

```diff
@@ -1,16 +1,19 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es6",
-        "outDir": "lib",
-        "lib": [
-            "es6"
-        ],
-        "declaration": true,
-        "rootDir": "src"
-    },
-    "exclude": [
-        "node_modules",
-        "lib"
-    ]
-}
\ No newline at end of file
+  "compilerOptions": {
+    "module": "es6",
+    "moduleResolution": "node",
+    "strict": true,
+    "target": "es5",
```

Comment:
Should the target have changed especially since the module is es6?

Reply:
This file was generated by autorest, so I'd rather just trust them to pick this stuff